### DISCUSSION TITLE
[Feature Request]: Ability to turn off new PHP version notifications.

### DISCUSSION THREAD

**Discussion by donsauk** [beyondcode/herd-community#730] [Ideas]
Version: Herd 1.10
OS: Windows 10

When running Herd on Windows 10, I get a notification every hour to update my PHP version. What if I want to keep my current PHP version? Now there are notifications for that non-stop and it is **VERY** annoying. Please let us modify the frequency of these notifications or let us turn it off completely.
![image](https://github.com/beyondcode/herd-community/assets/36240450/45ee8f0a-3a20-4dae-ac47-27135fc472c9)

**Comment by zilvmock** [comment]
I find this very annoying too!

**Reply by mpociot** [reply]
Sorry, will be fixed so that you get the notification only once - not every hour!
